apex/transformer/parallel_state.py

```diff
@@ -123,12 +138,11 @@ def initialize_model_parallel(
     ranks 8 to 15 belong to the second box.
     """
     # Get world size and rank. Ensure some consistencies.
+    p2p_backend = "nccl"
```

Comment:
we should remove this line at some point - this should respect the command line flag